OpenOversight/app/templates/input_find_officer.html

```diff
@@ -23,102 +22,72 @@
     </div>
     <form action="{{ url_for("main.get_officer") }}" method="post" class="form">
       {{ form.hidden_tag() }}
-      <div class="row form-group">
-        <div class="col-xs-12">
-          <ul class="nav nav-pills nav-justified thumbnail setup-panel">
-            <li class="active">
-              <a href="#step-1">
-                <h4 class="list-group-item-heading">Step 1</h4>
-                <p class="list-group-item-text">Department Information</p>
-              </a>
-            </li>
-            <li class="disabled">
-              <a href="#step-2">
-                <h4 class="list-group-item-heading">Step 2</h4>
-                <p class="list-group-item-text">Partial Name or Badge Number</p>
-              </a>
-            </li>
-            <li class="disabled">
-              <a href="#step-3">
-                <h4 class="list-group-item-heading">Step 3</h4>
-                <p class="list-group-item-text">Officer Rank</p>
-              </a>
-            </li>
-            <li class="disabled">
-              <a href="#step-4">
-                <h4 class="list-group-item-heading">Step 4</h4>
-                <p class="list-group-item-text">Officer Demographics</p>
-              </a>
-            </li>
-          </ul>
-        </div>
-      </div>
-      <div class="row setup-content"
-           id="step-1"
+      <div id="step-1"
+           class="card mb-5"
```

Comment:
Replacing the step-by-step dialogue in favor with a fully laid-out form since the old version was broken by the upgrade and this reduces user clicks and increases form completion